pythran/pythonic/numpy/random/bytes.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef PYTHONIC_NUMPY_RANDOM_BYTES_HPP
+#define PYTHONIC_NUMPY_RANDOM_BYTES_HPP
+
+#include "pythonic/include/numpy/random/bytes.hpp"
+#include "pythonic/include/numpy/random/generator.hpp"
+
+#include "pythonic/types/str.hpp"
+#include "pythonic/utils/proxy.hpp"
+
+#include <random>
+
+namespace pythonic
+{
+  namespace numpy
+  {
+    namespace random
+    {
+
+      types::str bytes(long length)
+      {
+        types::str result;
+        result.reserve(length);
+        std::uniform_int_distribution<char> distribution{0, 256};
+        for(long i=0; i<length; i++)
+          result.push_back(distribution(details::generator));
```

Comment:
Yes, So there will be an initialization for every values. I don't know what is the more efficient between one full useless initialisation and some push_back call with a reserve (and with this old computer, my measures are not very accurate). Did you do this measure?